packages/types/src/options.ts

```diff
@@ -211,6 +211,13 @@ export interface ClientOptions<TO extends BaseTransportOptions = BaseTransportOp
     [key: string]: any;
   };
 
+  /**
+   * Only works for Node or backend SDKs as we need Spotlight integration separately to keep the
+   * bundle tree-shakable. Adding here so we can warn if someone sets this to `true` on the browser
+   * SDKs, we can warn them about the "right way" (through integrations)
+   */
+  spotlight?: boolean | string;
```

Comment:
h: We shouldn't add this option globally to the SDK options as they're shared by browser and Node SDKs. The Node SDK already has the `spotlight` option declared: https://github.com/getsentry/sentry-javascript/blob/0d6647544eb4208da941ced0b89a84bd299c82f0/packages/node/src/types.ts#L60-L70

Reply:
See https://github.com/getsentry/sentry-javascript/pull/13263/files#r1707137593